shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -490,6 +503,59 @@ - (void)dealloc {
   [super dealloc];
 }
 
+#pragma mark - Status Bar touch event related swizzling
+
+static constexpr CGFloat kStandardStatusBarHeight = 20.0;
+
+static IMP __original_sendevent = nil;
+
+void _flutter_sendevent(id self, SEL _cmd, UIEvent *event) {
+  // Call the original implementation.
+  ((void(*)(id,SEL,UIEvent*))__original_sendevent)(self, _cmd, event);
+
+  // Intercept touchBegan events.
+  UITouch *touch = event.allTouches.anyObject;
```

Comment:
This check is incomplete. If one finger is already down and a second finger taps down, you may end up check the touch phase on a random touch.

I am not even sure taps with multiple fingers are a thing. I would just check if there is one touch event and its phase is `TouchPhaseBegan`.